backend/config/locales/zu.yml

```diff
@@ -137,13 +139,20 @@ zu:
       project_developers_in_project: Project developers in the project (%{count})
       index:
         priority_landscape: Priority Landscape
-      edit:
+      form:
```

Comment:
Hmm, I'm not sure what will happen with already translated strings in transifex.

Reply:
But here was a bug as looks like it must be either partial or view name, to use lazy loading i18n feature.

Reply:
Or we could not use lazy loading for this :). I think it will be need to be translated again...